tests/test_scheduler.py

```diff
@@ -317,3 +318,33 @@ def test_ssl_connection_scheduler(self):
         assert 'ssl_keyfile' not in redis_client.connection_pool.connection_kwargs
         assert 'ssl_certfile' not in redis_client.connection_pool.connection_kwargs
         assert 'ssl_ca_certs' not in redis_client.connection_pool.connection_kwargs
+
+
+class RedBeatLockTimeoutDefaultValues(RedBeatCase):
+    def test_no_values(self):
+        scheduler = RedBeatScheduler(app=self.app)
+        assert DEFAULT_MAX_INTERVAL * 5 == scheduler.lock_timeout
```

Comment:
this makes the think 5 should be a constant, but that's unrelated to this PR.

Reply:
I agree, there are a few more test cases that use `5` to test the timeout behavior.